plugins/lua/cxxrandom.lua

```diff
@@ -151,7 +151,7 @@ bool_distribution = {}
 function bool_distribution:new(chance)
     local o = {}
     self.__index = self
-    if type(min) ~= 'number' or type(max) ~= 'number' then
+    if type(chance) ~= 'number' then
         error("Invalid arguments in bool_distribution construction. min and max must be numbers.")
```

Comment:
Update the message.

Reply:
Yeah, was just about to comment about that